ImageViewer: Make arrow key shortcuts work

tcl3 opened the pull request:
The user can now navigate to the previous and next image using the left and right arrow keys respectively. These shortcuts were previously not working.

I've also renamed the actions from "Go Back" to "Go to Previous" and "Go Forward" to "Go to Next".

AtkinsSJ commented:
I don't think this is the right choice here. Those actions are called "go back" and "go forward" but they're really for navigating to the previous or next image, not going back or forward through your history. I think rather than reusing the forward/back CommonActions, these want to be renamed "Go to previous" and "Go to next".

tcl3 commented:
Hi @AtkinsSJ, thanks for your suggestions. I've updated the PR to make the left and right arrow keys work without the Alt modifier. I've also renamed the actions, as per your suggestion.

To get this to work I split the root widget into its own class, just to ignore all of its key events. This seems a bit heavy handed to me. Do you know if there's a more concise way to achieve the same thing?

AtkinsSJ commented:
Oops, sorry for taking so long to look at this! That approach seems reasonable. It seems heavy-handed now, but eventually it'll probably get converted to GML and then more of the code in `main.cpp` will move into the MainWidget.
